client/my-sites/earn/home.tsx

```diff
@@ -212,6 +213,57 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		};
 	};
 
+	/**
+	 * Return the content to display in the Donations card based on the current plan.
+	 *
+	 * @returns {object} Object with props to render a PromoCard.
+	 */
+	const getDonationsCard = () => {
+		const cta = isFreePlan
+			? {
+					text: translate( 'Unlock this feature' ),
+					action: () => {
+						trackUpgrade( 'any-paid-plan', 'donations' );
+						page( `/plans/${ selectedSiteSlug }` );
+					},
+			  }
+			: {
+					text: translate( 'Manage donations' ),
+					action: () => {
+						trackCtaButton( 'donations' );
+						page( `/earn/payments/${ selectedSiteSlug }` );
+					},
+			  };
+		const title = translate( 'Accept donations and tips' );
+
+		const body = (
+			<>
+				{ translate(
+					'Collect donations, tips, and contributions for your creative pursuits, organization, or whatever your website is about. '
+				) }
+				{ isFreePlan && <em>{ translate( 'Available with any paid plan' ) }.</em> }
```

Comment:
It'd be better if the final whitespace is left out of the translation. Otherwise translators might not include it in the translations thinking it's a mistake:
```suggestion
				{ translate(
					'Collect donations, tips, and contributions for your creative pursuits, organization, or whatever your website is about.'
				) }
				{ isFreePlan && <em>{ ' ' + translate( 'Available with any paid plan' ) }.</em> }
```